src/bun.js/webcore.zig

```diff
@@ -121,7 +121,7 @@ fn confirm(globalObject: *JSC.JSGlobalObject, callframe: *JSC.CallFrame) callcon
     // *  Not relevant in a server context.
 
     switch (first_byte) {
-        '\n' => return .false,
+        '\n', '\r' => return .false,
```

Comment:
when this \r is read, we should ignore it and read the next character.

i think this would cause a bug if you called confirm twice and press return

Reply:
Changed